src/main/java/com/jhuguet/sb_taskv1/app/controller/GiftCertificateController.java

```diff
@@ -95,17 +97,26 @@ public EntityModel<Page<GiftCertificate>> getAll(@RequestParam(defaultValue = "0
      * @return Filtered and/or sorted List of GiftCertificates
      */
     @GetMapping("/getBy")
-    public EntityModel<Page<GiftCertificate>> getBy(@RequestParam String partOfNameOrDescription, @RequestParam String tagName,
-                                                    @RequestParam String nameOrDate, @RequestParam String order,
+    public EntityModel<Page<GiftCertificate>> getBy(@RequestParam String partOfNameOrDescription,
+                                                    @RequestParam String tagName, @RequestParam String nameOrDate,
+                                                    @RequestParam String order,
                                                     @RequestParam(defaultValue = "0") int page,
-                                                    @RequestParam(defaultValue = "3") int size) throws InvalidInputInformation, PageNotFound {
+                                                    @RequestParam(defaultValue = "3") int size) throws
+            InvalidInputInformation, PageNotFound {
         pageResponse.validateInput(size, page);
 
         Page<GiftCertificate> certificates = giftCertificateService.filterCertificates(tagName, partOfNameOrDescription,
                 nameOrDate, order, PageRequest.of(page, size));
 
-        return EntityModel.of(certificates, linkTo(methodOn(GiftCertificateController.class)
-                .getBy(partOfNameOrDescription, tagName, nameOrDate, order, page, size)).withSelfRel());
+        return EntityModel.of(certificates,
+                linkTo(methodOn(GiftCertificateController.class).getBy(partOfNameOrDescription, tagName, nameOrDate,
+                        order, page, size)).withSelfRel());
+    }
+
+    @GetMapping("/neworder")
+    public String signUp(Model model) {
```

Comment:
What is it?

Reply:
Method thought for front end, will be deleted